docs/api/browser-window.md

```diff
@@ -1491,7 +1491,7 @@ Returns `BrowserView | null` - an attached BrowserView. Returns `null` if none i
 **Note:** The BrowserView API is currently experimental and may change or be
 removed in future Electron releases.
 
-[blink-feature-string]: https://cs.chromium.org/chromium/src/third_party/WebKit/Source/platform/runtime_enabled_features.json5?l=70
+[blink-feature-string]: https://cs.chromium.org/chromium/src/third_party/blink/renderer/platform/runtime_enabled_features.json5?l=70
```

Comment:
Your change looks good. Thanks!

But it's unclear to me why this link is named `blink-feature-string`. Maybe while you're here you could rename it to something more meaningful like `runtime-enabled-features`?

Reply:
Yes, fixed this :)